### ISSUE TITLE
Logs should be in the output panel

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#39638]
Refs: #39572

Logs are read only and thus shuold be in the output panel imho
They could have a channel (or multiple) of their own which could be shown in the output dropdown.

- [ ] Show logs in the output panel
- [ ] An action in the output panel to open the log file

**Comment by isidorn** [comment]
Another issue with the current approach is having to manage editors. Here's what just happened to me: i was inspecting logs, and left the log editor open. I restart vscode with a different log flag to only show errors, vscode opens and I still see the old logs (due to the editor being persisted). Only after I while I figured I need to re-fatch my logs and the shown state is stale

**Comment by joaomoreno** [comment]
I actually like it better if we put them into the Output pane. 👍 

Also fixes #39644
